services/web-app/utils/use-query-state.js

```diff
@@ -95,6 +95,14 @@ export const useQueryState = (
     if (saveToStorage && value) {
       localStorage.setItem(`${router.pathname}:${key}`, value)
```

Comment:
Is this line needed given line 100 below?

Reply:
it's not! you're right. I'm going to remove these changes anyway since I already took care of this over on #260